pkg/features/kube_features.go

```diff
@@ -791,6 +791,14 @@ const (
 	// Allow users to specify whether to take nodeAffinity/nodeTaint into consideration when
 	// calculating pod topology spread skew.
 	NodeInclusionPolicyInPodTopologySpread featuregate.Feature = "NodeInclusionPolicyInPodTopologySpread"
+
+	// owner: @mimowo
+	// kep: http://kep.k8s.io/3329
+	// alpha: v1.25
+	//
+	// Allow users to specify handling of pod failures based on container exit codes
+	// and pod end state.
```

Comment:
```suggestion
	// and pod conditions.
```

Reply:
Somehow this got reverted.

Reply:
yeah, I had two versions on two PRs and reconciled the other way. Now as suggested.